examples/block-interface.js

```diff
@@ -0,0 +1,25 @@
+import * as Block from 'multiformats/block'
+import * as codec from '@ipld/dag-cbor'
+import { sha256 as hasher } from 'multiformats/hashes/sha2'
+
+const value = { hello: 'world' }
+
+// encode a block
+let block = await Block.encode({ value, codec, hasher })
+
+block.value // { hello: 'world' }
+block.bytes // Uint8Array
+block.cid   // CID() w/ sha2-256 hash address and dag-cbor codec
+
+console.log("Example block CID: " + block.cid.toString())
+
+// you can also decode blocks from their binary state
+let block2 = await Block.decode({ bytes: block.bytes, codec, hasher })
+
+// check for equivalency using cid interface
+console.log("Example block CID equal to decoded binary block: " + block.cid.equals(block2.cid))
+
+// if you have the cid you can also verify the hash on decode
+let block3 = await Block.create({ bytes: block.bytes, cid: block.cid, codec, hasher })
+
+
```

Comment:
```suggestion
let block3 = await Block.create({ bytes: block.bytes, cid: block.cid, codec, hasher })
```

extraneous newlines at end of file